src/analysisd/analysisd.c

```diff
@@ -97,6 +158,19 @@ static unsigned int hourly_events;
 static unsigned int hourly_syscheck;
 static unsigned int hourly_firewall;
 
+/* EPS limits struct */
+typedef struct _limits_t {
+    unsigned int eps;
+    unsigned int timeframe;
+    unsigned int max_eps;
+    unsigned int total_eps_buffer;
+    unsigned int current_cell;
+    unsigned int wait_counter;
```

Comment:
Since this variable it protected with another mutex, can we define it outside this structure?

Reply:
Done